tests/test_sort_plugin.py

```diff
@@ -52,6 +52,41 @@ def test_finding_blank_line_separated_block(filetab):
     )
 
 
+def test_finding_indented_block(filetab):
+    filetab.textwidget.insert(
+        "end",
+        """\
+[foo]
+    dingdingding
+    akuli
+    catris
+    banana
+
+blah blah
+""",
+    )
+
+    filetab.textwidget.mark_set("insert", "3.3")
+    filetab.event_generate("<<Menubar:Edit/Sort Lines>>")
+    assert filetab.textwidget.index("insert") == "3.3"  # not moved
```

Comment:
There's no code to ensure a specific cursor place, so in the future, someone might be annoyed by whatever it's doing and want to change the behavior

```suggestion
```